dev/NavigationView/NavigationViewAutomationPeer.cpp

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "common.h"
+#include "ResourceAccessor.h"
+#include "NavigationViewAutomationPeer.h"
+#include "NavigationView.h"
+#include "NavigationViewItemBase.h"
+#include "SharedHelpers.h"
+
+
+CppWinRTActivatableClassWithBasicFactory(NavigationViewAutomationPeer);
+
+NavigationViewAutomationPeer::NavigationViewAutomationPeer(winrt::NavigationView const& owner) :
+    ReferenceTracker(owner)
+{
+}
+
+winrt::IInspectable NavigationViewAutomationPeer::GetPatternCore(winrt::PatternInterface const& patternInterface)
+{
+    if (patternInterface == winrt::PatternInterface::Selection)
+    {
+        return *this;
+    }
+
+    return __super::GetPatternCore(patternInterface);
+}
+
+bool NavigationViewAutomationPeer::CanSelectMultiple()
+{
+    return false;
+}
+
+bool NavigationViewAutomationPeer::IsSelectionRequired()
+{
+    return false;
```

Comment:
>false [](start = 11, length = 5)

Really?

Reply:
Yeah, that is what ListView was doing via SelectorAutomationPeer so I kept it the same. Also there are tests that exercise deselection via API (aka setting the selected item to null).